File: summinmax.py
import math
import os
import random
import re
import sys

# Complete the miniMaxSum function below.
def miniMaxSum(arr):
    
    print(str(sum(arr)-max(arr))+" "+ str(sum(arr)-min(arr)))

if __name__ == '__main__':
    arr =  arr = [11,2,4,4,5,6,10,8,-12]

    miniMaxSum(arr)

File: summinmax.php
<?php

// Complete the miniMaxSum function below.
function miniMaxSum($arr) {

    
    $max=max($arr);
    $min=min($arr);
    $t=array_sum($arr);
   echo ($t-$max)." ".($t-$min);
}

$stdin = fopen("php://stdin", "r");

fscanf($stdin, "%[^\n]", $arr_temp);

$arr = array_map('intval', preg_split('/ /', $arr_temp, -1, PREG_SPLIT_NO_EMPTY));

miniMaxSum($arr);

fclose($stdin);